docs/.vuepress/sidebar/en.ts

```diff
@@ -104,6 +104,10 @@ export const enSidebar = sidebar({
               text: "Wagmi",
               link: "/build/tutorials/tooling-guides/wagmi.md",
             },
+            {
+              text: "viem",
```

Comment:
We should capitalize this for consistency
```suggestion
              text: "Viem",
```